doc-files/release-notes-25.md

```diff
@@ -0,0 +1,158 @@
+[JDK-8146479](https://bugs.openjdk.org/browse/JDK-8146479) | Scene is black after stage is restored (content changed while minimized) | scenegraph
+[JDK-8340322](https://bugs.openjdk.org/browse/JDK-8340322) | Update WebKit to 620.1 | web
+[JDK-8347937](https://bugs.openjdk.org/browse/JDK-8347937) | Canvas pattern test fails and crashes on WebKit 620.1 | web
+[JDK-8349924](https://bugs.openjdk.org/browse/JDK-8349924) | Additional WebKit 620.1 fixes from WebKitGTK 2.46.6 | web
+[JDK-8350284](https://bugs.openjdk.org/browse/JDK-8350284) | WebKit 620.1 crashes on startup on Windows x86 32-bit | web
+[JDK-8351264](https://bugs.openjdk.org/browse/JDK-8351264) | Some images don't load with WebKit 620.1 | web
+[JDK-8352162](https://bugs.openjdk.org/browse/JDK-8352162) | Update libxml2 to 2.13.8 | web
+[JDK-8352164](https://bugs.openjdk.org/browse/JDK-8352164) | Update libxslt to 1.1.43 | web
+[JDK-8353916](https://bugs.openjdk.org/browse/JDK-8353916) | Unexpected event type for DOM mutation events with WebKit 620.1 | web
+[JDK-8354876](https://bugs.openjdk.org/browse/JDK-8354876) | Update SQLite to 3.49.1 | web
+[JDK-8354940](https://bugs.openjdk.org/browse/JDK-8354940) | Fail to sign in to Microsoft sites with WebView | web
+[JDK-8176813](https://bugs.openjdk.org/browse/JDK-8176813) | Mac: Failure to exit full-screen programmatically in some cases | window-toolkit
+[JDK-8348095](https://bugs.openjdk.org/browse/JDK-8348095) | [Linux] Menu shows up in wrong position when using i3 windows manager in full screen mode | window-toolkit
+[JDK-8351733](https://bugs.openjdk.org/browse/JDK-8351733) | Crash when creating too many nested event loops | window-toolkit
+[JDK-8353314](https://bugs.openjdk.org/browse/JDK-8353314) | macOS: Inconsistent fullscreen behavior | window-toolkit
+[JDK-8353548](https://bugs.openjdk.org/browse/JDK-8353548) | [macOS] DragEvent.getScreenY() returns incorrect value in secondary monitor | window-toolkit
+[JDK-8354478](https://bugs.openjdk.org/browse/JDK-8354478) | Improve StageStyle documentation | window-toolkit
+[JDK-8354631](https://bugs.openjdk.org/browse/JDK-8354631) | [macos] OpenURIHandler events not received by AWT when JavaFX is primary toolkit | window-toolkit
+[JDK-8356652](https://bugs.openjdk.org/browse/JDK-8356652) | Input field ignores custom input source characters | window-toolkit
+[JDK-8357584](https://bugs.openjdk.org/browse/JDK-8357584) | [XWayland] [OL10] Robot.mousePress() is delivered to wrong place | window-toolkit
+[JDK-8359601](https://bugs.openjdk.org/browse/JDK-8359601) | Fix window button states of an extended stage | window-toolkit
+[JDK-8359763](https://bugs.openjdk.org/browse/JDK-8359763) | Close request handler is not called for an extended stage | window-toolkit
+[JDK-8362095](https://bugs.openjdk.org/browse/JDK-8362095) | HeaderButtonMetrics should not be used across toolkit boundary | window-toolkit
```

Comment:
[JDK-8359601](https://bugs.openjdk.org/browse/JDK-8359601), [JDK-8359763](https://bugs.openjdk.org/browse/JDK-8359763), and [JDK-8362095](https://bugs.openjdk.org/browse/JDK-8362095) are bugs that are introduced and fixed in the same JFX version, and should probably be removed from the release notes.